make/Makefile.container.mk

```diff
@@ -60,5 +60,10 @@ else
 	podman push ${QUAY_TAG}
 endif
 
+## container-multi-arch-push-kiali-quay: Pushes the Kiali multi-arch image to quay.
+container-multi-arch-push-kiali-quay: .prepare-kiali-image-files
+	@echo Pushing Kiali multi-arch image to ${QUAY_TAG} using docker buildx
+	docker buildx build --push $(foreach arch,${TARGET_ARCHS},--platform=linux/${arch}) $(foreach tag,${QUAY_TAG},--tag=${tag}) -f ${OUTDIR}/docker/Dockerfile-multi-arch ${OUTDIR}/docker
```

Comment:
Because `buildx` is an experimental feature, we must ensure the user is OK that we are going to enable it. Also, `buildx` may not exist in the user's `docker` installation. Therefore, add a dependent target to ensure the user has `docker buildx` available and that the user has explicitly enabled it:
```suggestion
# Ensure "docker buildx" is available and enabled. For more details, see: https://github.com/docker/buildx/blob/master/README.md
.ensure-docker-buildx:
	@if docker buildx version > /dev/null 2>&1 || DOCKER_CLI_EXPERIMENTAL="${DOCKER_CLI_EXPERIMENTAL}" docker buildx version > /dev/null 2>&1 ; then \
	  echo "'docker buildx' is available and enabled."; \
	else \
	  if ! DOCKER_CLI_EXPERIMENTAL="enabled" docker buildx version > /dev/null 2>&1 ; then \
	    echo "You do not have 'docker buildx'. Note that 'buildx' is only available in Docker 19.03+"; \
	  else \
	    echo "You do not have 'docker buildx' enabled by default. Set DOCKER_CLI_EXPERIMENTAL=enabled if you want to use it."; \
	  fi; \
	  exit 1; \
	fi

## container-multi-arch-push-kiali-quay: Pushes the Kiali multi-arch image to quay.
container-multi-arch-push-kiali-quay: .ensure-docker-buildx .prepare-kiali-image-files
	@echo Pushing Kiali multi-arch image to ${QUAY_TAG} using docker buildx
	DOCKER_CLI_EXPERIMENTAL="enabled" docker buildx build --push $(foreach arch,${TARGET_ARCHS},--platform=linux/${arch}) $(foreach tag,${QUAY_TAG},--tag=${tag}) -f ${OUTDIR}/docker/Dockerfile-multi-arch ${OUTDIR}/docker
```